core-library/src/main/java/org/silverpeas/core/admin/space/SpaceInstLazyDataLoader.java

```diff
@@ -76,19 +92,55 @@ public class SpaceInstLazyDataLoader implements Serializable {
     this.space = space;
   }
 
+  /**
+   * Copies data of given instance into the current one.
+   * <p>
+   *   Current data are lost (replaced by the copied ones).
+   * </p>
+   * <p>
+   *   The source data are loaded before the real copy.
+   * </p>
+   * @param other {@link SpaceInstLazyDataLoader} instance data to copy.
+   */
+  void copy(final SpaceInstLazyDataLoader other) {
+    final List<String> componentIdsToCopy;
+    final List<String> subSpaceIdsToCopy;
+    final List<SpaceProfileInst> spaceProfilesToCopy;
+    synchronized (other.mutex) {
+      // loading data if not yet done
+      if (!other.loaded) {
+        other.loaded = true;
+        other.load();
```

Comment:
the `other.loaded` attribute shouldn't be set at true by the `other.load()` method itself once the loading done?

Reply:
Hum  ... not so convinced. But ok